src/index.ts

```diff
@@ -29,10 +43,32 @@ export function runInContext(
   context: Context,
   options: Partial<IOptions> = {}
 ): Promise<Result> {
+  function getFirstLine(theProgram: Program) {
+    if (theProgram.body[0] && theProgram.body[0].type === 'ExpressionStatement') {
+      const firstLineOfProgram = theProgram.body[0] as ExpressionStatement
+      if (firstLineOfProgram.expression.type === 'Literal') {
+        const firstLineExpression = firstLineOfProgram.expression as Literal
+        if (!!firstLineExpression) {
+          return firstLineExpression.value
+        } else {
+          return undefined
```

Comment:
You can omit all the `return undefined` here as this function is small and it's obvious that the function returns quickly if the conditions are not met.